src/network/netdev/ipoib.c

```diff
@@ -56,6 +57,48 @@ static int netdev_ipoib_fill_message_create(NetDev *netdev, Link *link, sd_netli
         return 0;
 }
 
+int ipoib_set_netlink_message(Link *link, sd_netlink_message *m) {
+        int r;
+
+        assert(link);
+        assert(link->network);
+        assert(m);
+
+        r = sd_netlink_message_set_flags(m, NLM_F_REQUEST | NLM_F_ACK);
+        if (r < 0)
+                return log_link_debug_errno(link, r, "Could not set netlink flags: %m");
```

Comment:
mention the flags in the message?

Reply:
This should not fail here, and if it fails, the error cause is not related to the flags, as the function accepts any non-zero flags.